### ISSUE TITLE
Deletion of folders not working

### ISSUE THREAD

**Issue by HexPandaa** [rclone/rclone-webui-react#103]
When deleting a folder, a POST request to `/operations/purge` is made, however a colon is not added to the name of a non-local remote, so the deletion fails.

The json sent looks like this:
```json
{"fs":"myRemote","remote":"some/path"}
```

But should look like this:
```json
{"fs":"myRemote:","remote":"some/path"}
```

This bug comes from the rclone-api JS library and I hopefully fixed it with the following pull request : https://github.com/rclone/rclone-js-api/pull/2
A simple update of the library should fix this issue.

**Comment by negative0** [comment]
Thanks. I have left a comment in the PR.

**Comment by negative0** [comment]
I will now close this issue as the PR is already merged rclone/rclone-js-api#2
